cyndi_setup.sql

```diff
@@ -0,0 +1,32 @@
+--
+-- Intializes a Cyndi-like inventory schema and creates a view copied from a
+-- local inventory DB
+--
+
+CREATE SCHEMA IF NOT EXISTS inventory;
+
+CREATE EXTENSION IF NOT EXISTS dblink;
+
+-- This view should be queried instead
+CREATE OR REPLACE VIEW inventory.hosts AS
+SELECT
+    id,
+    account,
+    display_name,
+    created_on as created,
+    modified_on as updated,
+    stale_timestamp,
+    stale_timestamp + INTERVAL '1' DAY * 7 AS stale_warning_timestamp,
+    stale_timestamp + INTERVAL '1' DAY * 14 AS culled_timestamp,
+    tags,
+    system_profile_facts as system_profile
```

Comment:
This must match the view that cyndi defines. We unfortunately don't have any say in the column names (`account` is an unfortunate choice, as you'll see in https://github.com/RedHatInsights/compliance-backend/pull/723)

Reply:
@vkrizan we cannot alias account as account_number, as I've described because we don't have control over this view in other environments.

Reply:
Fair enough. Can we gain control of the view?

Reply:
All apps use the same app-connector which writes to the underlying DB and in some cases sets up the view itself. Only in devel do we get to create the view ourselves, unfortunately. Not having control of the view is a feature of Cyndi to ensure consistency across apps.